### ISSUE TITLE
Incorrect image export

### ISSUE THREAD

**Issue by webghostx** [darktable-org/darktable#15520]
### Describe the bug

Suddenly there are problems when exporting JPEGS. It is a sequence of several almost identical images. All have been edited the same and have the exact same history stack. Surprisingly, however, the result is not identical. You can see the problem in the screenshot. Some pictures are good, others are very strange.

I don't know how to proceed to locate the problem. The terminal doesn't give me any errors
```
yves@swift3:~$ darktable -d verbose
    32.3176 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01695.ARW__v0.jpg'
    44.3149 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01696.ARW__v0.jpg'
    56.3965 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01697.ARW__v0.jpg'
    68.4038 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01698.ARW__v0.jpg'
    80.6686 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01699.ARW__v0.jpg'
    93.2485 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01700.ARW__v0.jpg'
   105.7703 [export_job] exported to `/home/yves/Bilder/Export-tmp/23.10.28_Mondfinsternis-Sequenz-6_A7_01701.ARW__v0.jpg'

```

### Steps to reproduce

difficult because it doesn't happen with all images

### Logfile | Screenshot | Screencast

![111-3](https://github.com/darktable-org/darktable/assets/5307244/8695e69f-6090-411d-9398-ee8c20c6c676)
![111](https://github.com/darktable-org/darktable/assets/5307244/83a4de29-9cbe-45fa-a0d8-fc15441b86a8)
![111-2](https://github.com/darktable-org/darktable/assets/5307244/27cd024c-4a64-43a2-8e0e-5423e123d1b8)

### Where did you install darktable from?

distro packaging

### darktable version

4.4.2

### What OS are you using?

Linux

### What is the version of your OS?

 Ubuntu 22.04.3 LTS   

### Describe your system?

yves@swift3:~$ darktable --version
this is darktable 4.4.2
copyright (c) 2009-2023 johannes hanika
https://github.com/darktable-org/darktable/issues/new/choose

compile options:
  bit depth is 64 bit
  normal build
  SSE2 optimized codepath enabled
  OpenMP support enabled
  OpenCL support enabled
  Lua support enabled, API version 9.1.0                                                                                                           
  Colord support enabled                                                                                                                           
  gPhoto2 support enabled                                                                                                                          
  GraphicsMagick support enabled                                                                                                                   
  ImageMagick support disabled                                                                                                                     
  libavif support enabled                                                                                                                          
  libheif support disabled                                                                                                                         
  libjxl support disabled                                                                                                                          
  OpenJPEG support enabled                                                                                                                         
  OpenEXR support enabled                                                                                                                          
  WebP support enabled   
                                                                                                                          
yves@swift3:~$ lsb_release -a                                                                                                                      
No LSB modules are available.                                                                                                                      
Distributor ID: Ubuntu                                                                                                                             
Description:    Ubuntu 22.04.3 LTS                                                                                                                 
Release:        22.04                                                                                                                              
Codename:       jammy 

yves@swift3:~$ lspci -nnk | grep -i VGA -A2
00:02.0 VGA compatible controller [0300]: Intel Corporation Iris Plus Graphics G4 (Ice Lake) [8086:8a5a] (rev 07)
        Subsystem: Acer Incorporated [ALI] Iris Plus Graphics G4 (Ice Lake) [1025:141a]
        Kernel driver in use: i915

### Are you using OpenCL GPU in darktable?

None

**Comment by jenshannoschwalm** [comment]
@webghostx would be interested to know if this also happens after upgrading to 4.6

If so, please report with a log using `-d pipe -d imageio`

**Comment by jenshannoschwalm** [comment]
Closing this for now as a) still insufficient log data and b) report on an old dt version with lots of fixes for 4.6.
